File: Rock Paper Scissor/Rock-Paper-Scissor.js
const Choices = ["rock","paper","scissor"];

while(true){
    var i = Math.floor(Math.random()*3);
    var Computer = Choices[i];
    var Player = "none";
    while(Player != "rock" && Player != "scissor" && Player != "paper"){
        Player = prompt("rock, paper, scissor? : ").toLowerCase(); 
    }
    if(Player == Computer){
        console.log("Computer's Choice : " + Computer);
        console.log("Your Choice : "+Player);
        console.log("Tie!!");
    }
    else if(Player == "rock"){
        if(Computer == "scissor"){
            console.log("Computer's Choice : " + Computer);
            console.log("Your Choice : "+Player);
            console.log("You Won!");
        }
        else{
            console.log("Computer's Choice : " + Computer);
            console.log("Your Choice : "+Player);
            console.log("You Lose!");
        }
    }
    else if(Player == "scissor"){
        if(Computer == "paper"){
            console.log("Computer's Choice : " + Computer);
            console.log("Your Choice : "+Player);
            console.log("You Won!");
        }
        else{
            console.log("Computer's Choice : " + Computer);
            console.log("Your Choice : "+Player);
            console.log("You Lose!");
        }
    }
    else if(Player == "paper"){
        if(Computer == "rock"){
            console.log("Computer's Choice : " + Computer);
            console.log("Your Choice : "+Player);
            console.log("You Won!");
        }
        else{
            console.log("Computer's Choice : " + Computer);
            console.log("Your Choice : "+Player);
            console.log("You Lose!");
        }
    }
    var response = prompt("Do you want to play again? (yes/no) : ").toLowerCase();

    if( response != "yes"){
        break;
    }      
    
}

File: Rock Paper Scissor/Rock-Paper-Scissor.py
 import random

game = ["rock","paper","scissor"]

while True:
  person = None
  computer = random.choice(game)

  while person not in game:
    person = input("rock, paper, scissor? : ").lower()

  if person == computer:
    print("Computer's Choice : " + computer)
    print("Your Choice : "+person)
    print("Tie!!")

  elif person == "rock":
    if computer == "paper":
      print("Computer's Choice : "+computer)
      print("Your Choice : "+person)
      print("You Lose!!")

    elif computer == "scissor":
      print("Computer's Choice : "+computer)
      print("Your Choice : "+person)
      print("You Won!!")

  elif person == "paper":
    if computer == "scissor":
      print("Computer's Choice : "+computer)
      print("Your Choice : "+person)
      print("You Lose!!")

    elif computer == "rock":
      print("Computer's Choice : "+computer)
      print("Your Choice : "+person)
      print("You Won!!")

  elif person == "scissor":
    if computer == "rock":
      print("Computer's Choice : "+computer)
      print("Your Choice : "+person)
      print("You Lose!!")

    elif computer == "paper":
      print("Computer's Choice : "+computer)
      print("Your Choice : "+person)
      print("You Won!!")

  response = input("Do you want to play again? (yes/no) : ").lower()

  if response != "yes":
    break
